### PR TITLE
Increase disk size of large postgres plans

### PR DESCRIPTION
**Pull request by Krenair**
What
----

Per https://github.com/alphagov/paas-team-manual/pull/415

How to review
-------------

Try it out in a dev env:
* Try upgrading from a medium high-iops plan to a old large plan, expect failure.
* Try upgrading from a medium high-iops plan to a new large plan, expect success.
* Try upgrading from an old large plan to a new large plan, expect success.
* Try creating an instance with the old plan, expect failure.

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨

### CODE REVIEW

File: manifests/cf-manifest/operations.d/720-rds-broker-postgres12-plans.yml [hunk 1]
```diff
@@ -91,9 +91,12 @@
     medium_plan_rds_properties: &medium_plan_rds_properties
       db_instance_class: "db.m5.large"
       allocated_storage: 100
-    large_plan_rds_properties: &large_plan_rds_properties
+    old_large_plan_rds_properties: &old_large_plan_rds_properties
```
Comment (LeePorte) [inline manifests/cf-manifest/operations.d/720-rds-broker-postgres12-plans.yml:94]: Should we change this to be `deprecated_large_plan_rds_properties` to make it linguistically consistent

File: manifests/cf-manifest/operations.d/720-rds-broker-postgres12-plans.yml [hunk 2]
```diff
@@ -260,16 +291,45 @@
         - "Dedicated Postgres 12 server"
         - "Storage Encrypted"
         - "AWS RDS"
+    rds_properties:
+      <<: *default_postgres_rds_properties # yamllint disable-line
+      <<: *old_large_plan_rds_properties # yamllint disable-line
```
Comment (LeePorte) [inline manifests/cf-manifest/operations.d/720-rds-broker-postgres12-plans.yml:296]: Should we change this to be `deprecated_large_plan_rds_properties` to make it linguistically consistent

File: manifests/cf-manifest/operations.d/720-rds-broker-postgres12-plans.yml [hunk 3]
```diff
@@ -290,7 +350,7 @@
         - "AWS RDS"
     rds_properties:
       <<: *default_postgres_rds_properties # yamllint disable-line
-      <<: *large_plan_rds_properties # yamllint disable-line
+      <<: *old_large_plan_rds_properties # yamllint disable-line
```
Comment (LeePorte) [inline manifests/cf-manifest/operations.d/720-rds-broker-postgres12-plans.yml:353]: Should we change this to be `deprecated_large_plan_rds_properties` to make it linguistically consistent

File: manifests/cf-manifest/spec/manifest/rds_broker_spec.rb [hunk 4]
```diff
@@ -139,13 +139,20 @@
       it { expect(rds_properties["db_instance_class"]).to match(/^db\.[a-z0-9]+\.large$/) }
     end
 
-    shared_examples "large sized plans" do
+    shared_examples "old large sized plans" do
```
Comment (LeePorte) [inline manifests/cf-manifest/spec/manifest/rds_broker_spec.rb:142]: Should we change this to be `deprecated large sized plans ` to make it linguistically consistent

File: manifests/cf-manifest/spec/manifest/rds_broker_spec.rb [hunk 5]
```diff
@@ -409,6 +422,17 @@
           it_behaves_like "Encryption enabled plans"
         end
 
+        describe "large-10-deprecated" do
+          subject(:plan) { pg_plans.find { |p| p["name"] == "large-10-deprecated" } }
+
+          it_behaves_like "all postgres plans"
+          it_behaves_like "postgres 10 plans"
+          it_behaves_like "old large sized plans"
```
Comment (LeePorte) [inline manifests/cf-manifest/spec/manifest/rds_broker_spec.rb:430]: `deprecated large sized plans`

File: manifests/cf-manifest/spec/manifest/rds_broker_spec.rb [hunk 6]
```diff
@@ -420,6 +444,17 @@
           it_behaves_like "Encryption enabled plans"
         end
 
+        describe "large-ha-10-deprecated" do
+          subject(:plan) { pg_plans.find { |p| p["name"] == "large-ha-10-deprecated" } }
+
+          it_behaves_like "all postgres plans"
+          it_behaves_like "postgres 10 plans"
+          it_behaves_like "old large sized plans"
```
Comment (LeePorte) [inline manifests/cf-manifest/spec/manifest/rds_broker_spec.rb:452]: As above

File: manifests/cf-manifest/spec/manifest/rds_broker_spec.rb [hunk 7]
```diff
@@ -617,6 +652,17 @@
           it_behaves_like "Encryption enabled plans"
         end
 
+        describe "large-11-deprecated" do
+          subject(:plan) { pg_plans.find { |p| p["name"] == "large-11-deprecated" } }
+
+          it_behaves_like "all postgres plans"
+          it_behaves_like "postgres 11 plans"
+          it_behaves_like "old large sized plans"
```
Comment (LeePorte) [inline manifests/cf-manifest/spec/manifest/rds_broker_spec.rb:660]: ditto

File: manifests/cf-manifest/spec/manifest/rds_broker_spec.rb [hunk 8]
```diff
@@ -628,6 +674,17 @@
           it_behaves_like "Encryption enabled plans"
         end
 
+        describe "large-ha-11-deprecated" do
+          subject(:plan) { pg_plans.find { |p| p["name"] == "large-ha-11-deprecated" } }
+
+          it_behaves_like "all postgres plans"
+          it_behaves_like "postgres 11 plans"
+          it_behaves_like "old large sized plans"
```
Comment (LeePorte) [inline manifests/cf-manifest/spec/manifest/rds_broker_spec.rb:682]: ditto

File: manifests/cf-manifest/spec/manifest/rds_broker_spec.rb [hunk 9]
```diff
@@ -825,6 +882,17 @@
           it_behaves_like "Encryption enabled plans"
         end
 
+        describe "large-12-deprecated" do
+          subject(:plan) { pg_plans.find { |p| p["name"] == "large-12-deprecated" } }
+
+          it_behaves_like "all postgres plans"
+          it_behaves_like "postgres 12 plans"
+          it_behaves_like "old large sized plans"
```
Comment (LeePorte) [inline manifests/cf-manifest/spec/manifest/rds_broker_spec.rb:890]: ditto

File: manifests/cf-manifest/spec/manifest/rds_broker_spec.rb [hunk 10]
```diff
@@ -836,6 +904,17 @@
           it_behaves_like "Encryption enabled plans"
         end
 
+        describe "large-ha-12-deprecated" do
+          subject(:plan) { pg_plans.find { |p| p["name"] == "large-ha-12-deprecated" } }
+
+          it_behaves_like "all postgres plans"
+          it_behaves_like "postgres 12 plans"
+          it_behaves_like "old large sized plans"
```
Comment (LeePorte) [inline manifests/cf-manifest/spec/manifest/rds_broker_spec.rb:912]: ditto

File: manifests/cf-manifest/spec/manifest/rds_broker_spec.rb [hunk 11]
```diff
@@ -1181,7 +1260,7 @@
           subject(:plan) { my_plans.find { |p| p["name"] == "large-unencrypted-5.7" } }
 
           it_behaves_like "all mysql 5.7 plans"
-          it_behaves_like "large sized plans"
+          it_behaves_like "old large sized plans"
```
Comment (LeePorte) [inline manifests/cf-manifest/spec/manifest/rds_broker_spec.rb:1263]: ditto

File: manifests/cf-manifest/spec/manifest/rds_broker_spec.rb [hunk 12]
```diff
@@ -1191,7 +1270,7 @@
           subject(:plan) { my_plans.find { |p| p["name"] == "large-5.7" } }
 
           it_behaves_like "all mysql 5.7 plans"
-          it_behaves_like "large sized plans"
+          it_behaves_like "old large sized plans"
```
Comment (LeePorte) [inline manifests/cf-manifest/spec/manifest/rds_broker_spec.rb:1273]: ditto

File: manifests/cf-manifest/spec/manifest/rds_broker_spec.rb [hunk 13]
```diff
@@ -1201,7 +1280,7 @@
           subject(:plan) { my_plans.find { |p| p["name"] == "large-ha-unencrypted-5.7" } }
 
           it_behaves_like "all mysql 5.7 plans"
-          it_behaves_like "large sized plans"
+          it_behaves_like "old large sized plans"
```
Comment (LeePorte) [inline manifests/cf-manifest/spec/manifest/rds_broker_spec.rb:1283]: ditto

File: manifests/cf-manifest/spec/manifest/rds_broker_spec.rb [hunk 14]
```diff
@@ -1211,7 +1290,7 @@
           subject(:plan) { my_plans.find { |p| p["name"] == "large-ha-5.7" } }
 
           it_behaves_like "all mysql 5.7 plans"
-          it_behaves_like "large sized plans"
+          it_behaves_like "old large sized plans"
```
Comment (LeePorte) [inline manifests/cf-manifest/spec/manifest/rds_broker_spec.rb:1293]: ditto

File: manifests/cf-manifest/spec/manifest/rds_broker_spec.rb [hunk 15]
```diff
@@ -1411,7 +1490,7 @@
           subject(:plan) { my_plans.find { |p| p["name"] == "large-8.0" } }
 
           it_behaves_like "all mysql 8.0 plans"
-          it_behaves_like "large sized plans"
+          it_behaves_like "old large sized plans"
```
Comment (LeePorte) [inline manifests/cf-manifest/spec/manifest/rds_broker_spec.rb:1493]: ditto

File: manifests/cf-manifest/spec/manifest/rds_broker_spec.rb [hunk 16]
```diff
@@ -1421,7 +1500,7 @@
           subject(:plan) { my_plans.find { |p| p["name"] == "large-ha-8.0" } }
 
           it_behaves_like "all mysql 8.0 plans"
-          it_behaves_like "large sized plans"
+          it_behaves_like "old large sized plans"
```
Comment (LeePorte) [inline manifests/cf-manifest/spec/manifest/rds_broker_spec.rb:1503]: ditto

File: manifests/cf-manifest/operations.d/716-rds-broker-postgres11-plans.yml [hunk 17]
```diff
@@ -290,7 +350,7 @@
         - "AWS RDS"
     rds_properties:
       <<: *default_postgres_rds_properties # yamllint disable-line
-      <<: *large_plan_rds_properties # yamllint disable-line
+      <<: *old_large_plan_rds_properties # yamllint disable-line
```
Comment (LeePorte) [inline manifests/cf-manifest/operations.d/716-rds-broker-postgres11-plans.yml:353]: Should we change this to be `deprecated_large_plan_rds_properties` to make it linguistically consistent

File: manifests/cf-manifest/operations.d/716-rds-broker-postgres11-plans.yml [hunk 18]
```diff
@@ -91,9 +91,12 @@
     medium_plan_rds_properties: &medium_plan_rds_properties
       db_instance_class: "db.m5.large"
       allocated_storage: 100
-    large_plan_rds_properties: &large_plan_rds_properties
+    old_large_plan_rds_properties: &old_large_plan_rds_properties
```
Comment (LeePorte) [inline manifests/cf-manifest/operations.d/716-rds-broker-postgres11-plans.yml:94]: Should we change this to be `deprecated_large_plan_rds_properties` to make it linguistically consistent

File: manifests/cf-manifest/operations.d/715-rds-broker-postgres10-plans.yml [hunk 19]
```diff
@@ -290,7 +350,7 @@
         - "AWS RDS"
     rds_properties:
       <<: *default_postgres_rds_properties # yamllint disable-line
-      <<: *large_plan_rds_properties # yamllint disable-line
+      <<: *old_large_plan_rds_properties # yamllint disable-line
```
Comment (LeePorte) [inline manifests/cf-manifest/operations.d/715-rds-broker-postgres10-plans.yml:353]: Should we change this to be `deprecated_large_plan_rds_properties` to make it linguistically consistent

File: manifests/cf-manifest/operations.d/715-rds-broker-postgres10-plans.yml [hunk 20]
```diff
@@ -260,16 +291,45 @@
         - "Dedicated Postgres 10 server"
         - "Storage Encrypted"
         - "AWS RDS"
+    rds_properties:
+      <<: *default_postgres_rds_properties # yamllint disable-line
+      <<: *old_large_plan_rds_properties # yamllint disable-line
```
Comment (LeePorte) [inline manifests/cf-manifest/operations.d/715-rds-broker-postgres10-plans.yml:296]: Should we change this to be `deprecated_large_plan_rds_properties` to make it linguistically consistent

File: manifests/cf-manifest/operations.d/715-rds-broker-postgres10-plans.yml [hunk 21]
```diff
@@ -91,9 +91,12 @@
     medium_plan_rds_properties: &medium_plan_rds_properties
       db_instance_class: "db.m4.large"
       allocated_storage: 100
-    large_plan_rds_properties: &large_plan_rds_properties
+    old_large_plan_rds_properties: &old_large_plan_rds_properties
```
Comment (LeePorte) [inline manifests/cf-manifest/operations.d/715-rds-broker-postgres10-plans.yml:94]: Should we change this to be `deprecated_large_plan_rds_properties` to make it linguistically consistent

**Review by LeePorte — CHANGES_REQUESTED** [review]
Small wording change to make things linguistically consistent

File: manifests/cf-manifest/operations.d/720-rds-broker-postgres12-plans.yml (hunk 1 above)
Comment (Krenair) [reply manifests/cf-manifest/operations.d/720-rds-broker-postgres12-plans.yml:94]: Going to keep this as old to be consistent with the old_small stuff, and change `-deprecated` to `-old` in the names

**Review by LeePorte — APPROVED** [review]

**Pull request merged**